packages/eas-cli/src/commands/secret/create.ts

```diff
@@ -121,9 +120,7 @@ export default class EnvironmentSecretCreate extends Command {
       }
 
       Log.withTick(
-        `️Created a new secret ${chalk.bold(name)} on project ${chalk.bold(
-          `@${accountName}/${slug}`
-        )}.`
+        `️Created a new secret ${chalk.bold(name)} on project with id ${chalk.bold(projectId)}.`
```

Comment:
this feels like a meaningful regression in the quality of the output. i don't think anyone will learn their project uuid and so it seems better to not output it, or to print some human readable identifier for the project and the uuid dimmed beside it. we should look at other clis to see how they handle this, nothing comes to mind off of the top of my head 😂

Reply:
good point